src/grb_model.jl

```diff
@@ -104,7 +106,7 @@ end
 
 # read / write file
 
-function read_model(model::Model, filename::ASCIIString)
+function read_model(model::Model, filename::String)
```

Comment:
the win32 unicode api's will usually be in utf16 though, so it depends what gurobi uses to manipulate files